pages/docs/couple-your-code/couple-your-code-implicit-coupling.md

```diff
@@ -58,7 +58,7 @@ while (precice.isCouplingOngoing()){
   preciceDt = precice.getMaxTimeStepSize();
   solverDt = beginTimeStep(); // e.g. compute adaptive dt
   dt = min(preciceDt, solverDt);
-  precice.readBlockVectorData(displID, vertexSize, vertexIDs, displacements);
+  precice.readBlockVectorData(displID, vertexSize, vertexIDs, preciceDt, displacements);
```

Comment:
```suggestion
  precice.readBlockVectorData(displID, vertexSize, vertexIDs, dt, displacements);
```
no?
and maybe we could add a comment that we are reading at the end of the time step?

Reply:
Fixed. 3635024f76cd3bd36cddd12e046db2de949cffc4